### ISSUE TITLE
DetailsList does not announce expand/collapsed state of entire group in Narrator

### ISSUE THREAD

**Issue by sereymonyros** [microsoft/fluentui#16130]
<DetailsList

Step to repro:
CodePen: [DetailsList Repro](https://codepen.io/justslone-the-encoder/pen/OJRNgea?editors=0010)
1) use keyboard narrator to access to the parent level of detailslist
2) hit enter/space (note you will need to exit scan mode to invoke the expand collapse)

Actual result: narrator doesn't speak collapse/expanded
Expected result: narrator should speak collapse/expanded

**Comment by JustSlone** [comment]
@sereymonyros thanks for filing this issue! I confirmed that narrator does not read the expand/collapsed state of the expand collapse group. I also updated the issue with a specific codepen, in the future please provide a codepen repro with issues you file. Also a gif showing the behavior is very helpful. 

On this issue itself, this looks like an issue specific to narrator. NVDA successfully reads the expand/collapse state. Also I confirmed that the element is using aria-expanded=true/false, so this one might be difficult to fix, it might be an issue in narrator. 
See this example with NVDA: 
![details-expand-nvda](https://user-images.githubusercontent.com/40304143/101266076-a86a5280-3700-11eb-834a-2ffde386503d.gif)

I welcome anyone who wants to dig in and see if there is a way we can fix this for narrator.

**Comment by JustSlone** [comment]
Minor update on this issue, we are making improvements and fixes to DetailsList accessibility generally, this issue may get fixed by some of the other improvements.

**Comment by JustSlone** [comment]
Confirmed it's still an issue with narrator
